evalml/pipelines/components/transformers/preprocessing/time_series_featurizer.py

```diff
@@ -258,7 +260,7 @@ def _compute_delays(self, X_ww, y):
         # Features created from categorical columns should no longer be categorical
         lagged_features = pd.DataFrame(lagged_features)
         lagged_features.ww.init(
-            logical_types={col: "Double" for col in cols_derived_from_categoricals},
+            logical_types={col: "Double" for col in lagged_features.columns},
```

Comment:
I'm not confident in the correctness of this change, as to how we handle all types and if everything does in fact become a double here. If anyone knows better about this, please let me know.